.github/workflows/ci.yml

```diff
@@ -67,9 +67,6 @@ jobs:
       - name: Check binary compatibility
         run: sbt ++${{ matrix.scala }} mimaReportBinaryIssues
 
-      - name: Check unused dependencies
-        run: sbt ++${{ matrix.scala }} unusedCompileDependenciesTest
-
```

Comment:
I'd like to keep this one.  It has saved us from some bugs in the past.  (In fact, I'd like to adopt `undeclaredCompileDependenciesTest`, which is verbose, but would have saved us from a cursed release recently.)

Reply:
I'll check out this branch and make the task pass.

Reply:
https://github.com/RaasAhsan/http4s/pull/1

Reply:
Thanks, there are some flaky tests that still need to be fixed, taking a look at those